pkg/providers/aws/provider_redis_test.go

```diff
@@ -1058,7 +1058,7 @@ func Test_createRedisCluster(t *testing.T) {
 			wantErr: true,
 		},
 		{
-			name: "test elasticache buildReplicationGroupPending is called (valid cluster rhmi subnets)",
+			name: "test elasticache buildReplicationGroupPending is called (valid cluster subnets)",
```

Comment:
don't think we need to change these test name as we will still have rhmi CR's in rhoam , if you want you can change them to rhoam subnets

Reply:
we can call them `bundled subnets` rather than specific product subnets, it just refers to the subnets that CRO used to create inside the the cluster vpc before we implemented the standalone vpc